app/src/main/res/values/strings.xml

```diff
@@ -1,81 +1,179 @@
     <string name="permission_location">Will you allow %1$s to access your location?</string>
+    <!-- LOCALIZATION NOTE (permission_notification): '%1$s' will be replaced at runtime with the app name.
+         The string 'permission_notification' will be displayed in a dialog box presented to the user
+         when the application needs to request permission to send system notifications. -->
     <!-- Parameter will be replaced with the app name -->
     <string name="permission_notification">Will you allow %1$s to send notifications?</string>
-    <!-- Parameter will be replaced with the app name -->
+    <!-- LOCALIZATION NOTE (permission_read_external_storage): '%1$s' will be replaced at runtime with the app name.
+         The string 'permission_read_external_storage' will be displayed in a dialog box presented to the user
+         when the application needs to request permission to access the device's external storage such as the
+         sdcard. -->
     <string name="permission_read_external_storage">Will you allow %1$s to read your external storage?</string>
+    <!-- LOCALIZATION NOTE (): FIXME It doesn't look like this string is actually used -->
     <string name="speak_now">Speak now</string>
+    <!-- LOCALIZATION NOTE (settings_version): '%1$s' will be replaced at runtime with the application
+         version number. The string 'settings_version' is displayed in the settings dialog box and how
+         the user can know what version of the application they are currently running. -->
     <string name="settings_version">version %1$s</string>
+    <!-- LOCALIZATION NOTE (settings_crash_reporting): The string 'settings_crash_reporting' is
+         displayed in the settings dialog and is used to label an ON/OFF switch which toggles whether
+         an application crash is reported with out prompting the user first. -->
     <string name="settings_crash_reporting">Crash Reporting</string>
+    <!-- LOCALIZATION NOTE (settings_telemetry): The string 'settings_telemetry' is displayed in the
+         settings dialog and is used to label an ON/OFF switch which toggles whether telemetry is
+         reported back to Mozilla. -->
     <string name="settings_telemetry">Telemetry</string>
+    <!-- LOCALIZATION NOTE (settings_privacy): The string 'settings_privacy' is used to label a button
+         that when pressed loads a web page the describes Mozilla's privacy policy. -->
     <string name="settings_privacy">Privacy Policy</string>
+    <!-- LOCALIZATION NOTE (settings_report_issue): The string 'settings_report_issue' is used to label
+         a button that when pressed loads a web page that allows the user file an issue or problem
+         they may have encountered while using the application. -->
     <string name="settings_report_issue">Report an issue</string>
+    <!-- LOCALIZATION NOTE (settings_version_developer): FIXME This string probably doesn't need to be localized/ -->
```

Comment:
IMO it still should be, as we can't make an assumption that devs/reports speak/read English